authenticator/authenticator.go

```diff
@@ -63,6 +63,9 @@ type Authenticator struct {
 	// TokenCache is used to retrieve previously acquired tokens and saving new ones
 	// If not set we do not use a cache to store the tokens
 	TokenCache TokenCache
+	// TokenExpiration defines lifetime of generated tokens
+	// Has no effect for tokens provided via the Token field
+	TokenExpiration string
```

Comment:
I would rather see this as an integer in seconds, so the authenticater internally converts it to a tokenexpiration string, with the format the api wants to see.

Reply:
Had hat initially, then found out the API needs an undocumented string format.

Took the safe way and just exposed it as an undocumented string, just like the API itself does.

To be more in the spirit of Go, I would rather have this field be a [time.Duration](https://golang.org/pkg/time/#Duration). Would the API accept anything [time.Duration.String()](https://golang.org/pkg/time/#Duration.String) outputs? Or is converting to seconds or minutes the best approach?

Reply:
time.Duration would be nice indeed :+1: 
The api documentation will be updated when it comes to the expiration_time field, this is good feedback, thanks.
Converting it to minutes is the best approach.

Reply:
I used time.Duration, but exported to seconds for more flexibility.